dist/Dockerfile.e2e/Dockerfile

```diff
@@ -24,6 +24,6 @@ ENV PATH="${PATH}:${HOME}/bin"
 
 COPY --from=builder /opt/cincinnati/bin/cincinnati-e2e-test /usr/bin
 COPY --from=builder hack/e2e.sh hack/
-COPY --from=builder dist/openshift/cincinnati.yaml template/
+COPY --from=builder dist/openshift/cincinnati.yaml dist/openshift/
```

Comment:
This change confuses me. Is there a change on CI to make this work as well?

Reply:
This line needs to be coupled with https://github.com/openshift/cincinnati/pull/244/files#diff-9dac1c15b19c2b8ae927c1c5b610307aR48

Reply:
Ah understood, thanks!